.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,43 @@
+name: Continuous Integration
+on:
+  push:
+    branches:
+    - master
+    tags:
+    - '*'
+  pull_request:
```

Comment:
this is only here bc i was testing it out, otherwise we should remove since we don't need to do this on pulls